batch-jberet/src/main/java/org/wildfly/extension/batch/jberet/deployment/JobOperatorService.java

```diff
@@ -488,6 +488,8 @@ private void stopRunningJobs(final boolean queueForRestart) {
                             }
                         }
                     }
+                } catch (IllegalStateException e) {
+                    BatchLogger.LOGGER.error(e);
```

Comment:
We probably want more information about the job here. We should add a new logger message like you did above with some information about the job.